roles/default/kiali-deploy/defaults/main.yml

```diff
@@ -188,6 +188,24 @@ kiali_defaults:
     web_root: ""
     web_schema: ""
 
+  health_config:
+    rate:
+      - namespace: ".*",
```

Comment:
it doesn't work for me, shouldn't be comas at the end of those lines.
I get the following error:

```
ERROR! Syntax Error while loading YAML.
  expected <block end>, but found ','

The error appears to be in '/opt/ansible/roles/default/kiali-deploy/defaults/main.yml': line 193, column 24, but may
be elsewhere in the file depending on the exact syntax problem.

The offending line appears to be:

    rate:
      - namespace: ".*",
                       ^ here

```